build.gradle

```diff
@@ -17,35 +19,39 @@ repositories {
 dependencies {
     implementation "com.github.node-gradle:gradle-node-plugin:${project.'gradle-node-plugin.version'}"
     api "org.octopusden.octopus.infrastructure:components-registry-service-client:${project['octopus-components-registry-service.version']}"
+    api "com.platformlib:platformlib-process-local:${project['platformlib-process.version']}"
```

Comment:
@adellinekania why this dependency has been added?

Reply:
This is not a new dependency; it was already in use but listed under a different section. I have now merged it into only one dependency section.